features/ProductNews.feature

```diff
@@ -0,0 +1,53 @@
+@owner:derek
+@author:lilla
+
+Feature: Product News Page
+
+    Display the Product News Page with cards to navigate to news.
+
+    @motivating
+    Scenario: Product news updates can be filtered by type
+        Given a user selects from the type dropdown
+        Then they can see the following items
+
+            | postType         |
+            | All Posts        |
+            | Feature Releases |
+            | Monthly Updates  |
+            | Product Updates  |
+
+    @motivating
+    Scenario: Product news updates can be filtered by date
+        Given a user selects from the date dropdown
+        Then they can see the following items
+
+            | date          |
+            | From All-time |
+            | Last 30 days  |
+            | Past Year     |
+
+    @motivating
+    Scenario: News cards are showing on the page in one column regardless of the screen size
+        Given a user looks at the card
+        Then they see the following items
+
+            | cardParts |
+            | Title     |
+            | Date      |
+            | Tags      |
+            | Content   |
+
+    @ui
+    Scenario: CTA button visibility
```

Comment:
I would suggest making this a scenario outline, to cover both cases of when something is configured for the CTA button and when something isn't